portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,243 @@
-#content {
+div {
+  position: relative;
+}
+
+#nav {
+  position: absolute;
+  left: 50%;
+  top: 0;
+  margin-left: -220px;
+  /* negative margin equal to half the width */
+  width: 750px;
+  list-style: none;
+}
+
+#nav li {
+  float: left;
+}
+
+#nav a {
+  border: none;
+  color: white;
+  padding: 15px 32px;
+  text-align: center;
+  text-decoration: none;
+  display: block;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+  font-family: 'Roboto', sans-serif;
+  background-color: rgb(0, 0, 0);
+}
+
+#pageA {
+  background-color: #d3deeb;
+}
+
+.icon {
+  border-radius: 50%;
+  margin-top: 70px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+}
+
+#name {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 80px;
+  text-align: center;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+#info {
+  font-family: 'Source Sans Pro', sans-serif;
+  font-size: 28px;
+  text-align: center;
+  margin-bottom: 10px;
+}
+
+#bio {
+  font-family: 'Roboto', sans-serif;
+  font-size: 15px;
+  text-align: center;
+}
+
+.fa {
+  padding: 20px;
+  font-size: 30px;
+  width: 20px;
+  text-align: center;
+  text-decoration: none;
+  margin: 5px 2px;
+  border-radius: 50%;
+  margin-top: 20px;
+}
+
+.fa:hover {
+  opacity: 0.7;
+}
+
+.fa-facebook {
+  background: black;
+  color: white;
+}
+
+.fa-twitter {
+  background: black;
+  color: white;
+}
+
+.fa-instagram {
+  background: black;
+  color: white;
+}
+
+.fa-linkedin {
+  background: black;
+  color: white;
+}
+
+.fa-github {
+  background: black;
+  color: white;
+}
+
+.pageB {
+  background-color: #d6cac4;
+}
+
+#facts {
+  font-family: 'Roboto', sans-serif;
+  font-size: 50px;
+  text-align: center;
+  margin-top: 70px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.list {
+  text-align: center;
+  margin-top: 70px;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  font-size: 20px;
+  font-family: 'Roboto', sans-serif;
+}
+
+.pageC {
```

Comment:
pageA, pageB, pageC are hard to interpret in general. Tommorrow if someone wanted to modify this code, they may require context of what pageA, pageB  and pageC are. Its a good practice to name them meaningfully.  No action required for now though.